integration/spark2/pom.xml

```diff
@@ -134,6 +134,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
Why is this dependency needed in spark2?

Reply:
Because when I use org.apache.carbondata.spark.thriftserver.CarbonThriftServe and use beeline to connect, it throw NoClassDefFoundError。NoClassDefFoundError: org/apache/http/impl/conn/PoolingClientConnectionManager。  PoolingClientConnectionManager is belong to httpclient, so I add:

    <dependency>
      <groupId>org.apache.httpcomponents</groupId>
      <artifactId>httpclient</artifactId>
      <version>${httpclient.version}</version>
    </dependency>

Reply:
But nobody else is facing this issue. And also none of our environments are getting this exception. Maybe u need to add httpclient jars to you cluster seperately?

Reply:
I meet more than two times. This issue also happen in spark on k8s before. You can try in original cluster. 
It's not convenient for user to  add httpclient jars to you cluster seperately.  What's effect if we add this dependency?

Reply:
I have never encounter this problem before, can you describe the scenario in more detail, in what case NoClassDefFoundError will be thrown?

Reply:
I see `org.apache.httpcomponents` is already introduced in other modules, please unify the version for this jar in all places

Reply:
ok, done